### DISCUSSION TITLE
specifying a controller

### DISCUSSION THREAD

**Discussion by pishguy** [spatie/laravel-route-attributes#22] [Q&A] [answered]
how we can specify `Controller` resource in controller class?

for example you suppose we want to make this route:

```php
Route::resource('/post',Post::class)->names('posts);
```

**Comment by freekmurze** [comment, marked as answer]
This currently isn't supported. Feel free to PR new features

**Comment by osbre** [comment]
Doing some progress about it right now:
https://github.com/spatie/laravel-route-attributes/pull/44

Update: released in 1.5.0
